lib/google-apis/auth-client.js

```diff
@@ -69,6 +79,31 @@ function AuthClient(projectId, shouldReportErrors, key) {
 }
 
 /**
+ * The init event is emitted when either the underlying transport has
+ * successfully initialized and authenticated or when it has failed to
+ * initialize or authenticate. The event is called with on argument which
+ * is a Boolean indicating the client initialization status. If false the
+ * client has failed to initialize and the instance will not be able to send
+ * requests. If true the client has successfully initialized and is ready to
+ * start sending requests.
+ * @event AuthClient#init
+ * @type {Boolean} - The initialization status of the client
+ */
+inherits(AuthClient, EventEmitter);
+
+/**
+ * Returns the requesting status of the client, indicating whether or not the
+ * client is currently requesting or not. Will return true if the client is
+ * currently requesting or false if the client is idle.
+ * @function getRequestStatus
+ * @return {Boolean} - the the requesting status of the client
+ */
+AuthClient.prototype.getRequestStatus = function() {
```

Comment:
This is public only so that tests could call it. It would be preferrable to mark/document this as being for tests only.